mesa/experimental/devs/simulator.py

```diff
@@ -298,17 +354,6 @@ def run_until(self, end_time: int) -> None:
                 self._schedule_event(event)
                 break
 
-    def run_for(self, time_delta: int):
```

Comment:
Why is `run_for` removed?

Reply:
because of the the t=0 to t=1 starting time. With the first step now on t=1, the old behavior is no longer needed and it can fall back on the behavior in `Simulator`.

Reply:
It’s probably nice to deprecate it.

Reply:
no, because the behavior is wrong now. 

The code was

```python
    def run_for(self, time_delta: int):
        """Run the simulator for the specified time delta.

        Args:
            time_delta (float| int): The time delta. The simulator is run from the current time to the current time
                                     plus the time delta

        """
        end_time = self.time + time_delta - 1
        self.run_until(end_time)
```

But the parent class has

```python
        end_time = self.time + time_delta
        self.run_until(end_time)
```

which is now correct behavior. So, nothing needs to be deprecated because the method is still there with the correct behavior on the parent class.

Reply:
Got it, thanks